test/new-e2e/pkg/environments/kindvmhost.go

```diff
@@ -0,0 +1,26 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package environments
+
+import (
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+)
+
+// KindVMHost is an environment that contains a Kubernetes cluster, a VM and a FakeIntake.
+type KindVMHost struct {
+	// Components
+	KubernetesCluster *components.KubernetesCluster
+	FakeIntake        *components.FakeIntake
+	Agent             *components.KubernetesAgent
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should we add the host too ? How is the `Agent` used ? Wonder if we should take it out, as we don't interact with it

Reply:
`Agent` is used because we export the name of the Helm release that installed the agent, I saw that the previous run function we had for Kind did this and I think containers test needs this information. But yes for now the Agent does not include a Client to interact with the Agent.
I can add the `Host`but I feel like we should not interact with the Host when the agent is running in Kubernetes. Because `Host` has a sense only for KindVM tests, if you are in EKS you do not have an Host